src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: fit-content;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
```

Comment:
The selector `&--4 &__star:nth-child(-n + 4)` is not following the BEM naming convention correctly. In BEM, modifiers should be applied directly to the block or element they modify, without nesting. Consider using a more appropriate BEM structure, such as `.stars--4 .stars__star:nth-child(-n + 4)`.